packages/ui-extensions-server-kit/src/typings.d.ts

```diff
@@ -1,2 +0,0 @@
-declare module '*.scss';
-declare module '*.css';
```

Comment:
There are no stylesheets in this library.

Reply:
There will be styles when the `componentization` branch is merged.